test/integration/targets/bigip_device_info/tasks/issue-01757.yaml

```diff
@@ -0,0 +1,11 @@
+---
+
+- name: Issue 01757 - Select all facts, single keyword
+  f5networks.f5_modules.bigip_device_info:
```

Comment:
why is this here? this should work without the FQCN to the collection

Reply:
For some reason it wasn't... I'll remove it, of course. Need to investigate my env and see why it's working...

Reply:
check the email
i sent on the dev env few days ago if this is all good and still broken ill test locally when i am back

Reply:
No no, it's all good... Somehow started working... I think it's because I logged into bash again...